### DISCUSSION TITLE
Nextjs With Typescrpit

### DISCUSSION THREAD

**Discussion by mehradopek** [vercel/next.js#14357] [Help]
anyone know an open source project with  nextjs and typescript?

**Comment by ddprrt** [comment]
We built the Script'19 website with it: https://github.com/stahlstadtjs/scriptconf.org -- https://scriptconf.org
Since it was my pet project the component structure is meh, but it was fun doing it with Next!

Also https://github.com/stahlstadtjs/scriptcast-app was done with Next + TypeScript

**Comment by wodka** [comment]
ohmyform also uses nextjs with typescript: https://github.com/ohmyform/ui
